.codespellignore

```diff
@@ -0,0 +1,2 @@
+caf
```

Comment:
bonus point if you merge a patch in tar to allow using `calf` as an alias to `caf` :)